src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -80,7 +80,7 @@ azure-mgmt-sqlvirtualmachine==1.0.0b5
 azure-mgmt-storage==21.0.0
 azure-mgmt-synapse==2.1.0b5
 azure-mgmt-trafficmanager==1.0.0
-azure-mgmt-web==7.0.0
+azure-mgmt-web==0.42.0
```

Comment:
I would remove the changes in the requirements.py and setup.py files since we will have them in a separate PR

Reply:
done!